bitwarden_license/src/CommCore/CommCore.csproj

```diff
@@ -2,6 +2,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\..\src\Core\Core.csproj" />
+    <ProjectReference Include="..\Commercial.Infrastructure.EntityFramework\Commercial.Infrastructure.EntityFramework.csproj" />
```

Comment:
Core shouldn't depend on infrastructure.

Reply:
resolved in https://github.com/bitwarden/server/pull/2144/commits/5d6a2145bcf5210ab3c7a59a440cfa06aa87f6d0